### PR TITLE
Merging Develop into main

### PR DESCRIPTION
**Pull request by kashan-01**
[PS-1]
	- Resolved conflicts
<br>

- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Every function, interface, class has a comment describing what it does and input/output parameters

Signed-off-by: Kashan Ali [ kashan.ali@xgrid.co ]

### CODE REVIEW

**Review by muhammad-irfan-xgrid — DISMISSED** [review]
LGTM

File: infrastructure/config.sh [hunk 1]
```diff
@@ -23,3 +23,4 @@ export domain="example.test.co"
 export owner_email="admin@test.co"
 export creator_email="admin@test.co"
 export namespace="example"
+export env="prod"
```
Comment (samanXgrid) [inline infrastructure/config.sh:26]: please add example here or dev here not prod

**Review by samanXgrid — APPROVED** [review]
LGTM

**Review by muhammad-irfan-xgrid — APPROVED** [review]
LGTM

**Review by mfaaraz — APPROVED** [review]
LGTM

**Review by yaseenzafar123 — APPROVED** [review]
LGTM

**Pull request merged**